_data/student_projects.yml

```diff
@@ -14,4 +14,13 @@
 - name: Mochammad Rafi Al Rizkan
   github: https://github.com/kol4k/eKantin
   author: Rafi
-  image: rafi.png
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Invalid or bad conflict resolution. Please take in more care while resolving conflicts.